src/models/oryp9/repairs.md

```diff
@@ -0,0 +1,252 @@
+
+Replacement heat-conductive metal fitted to the machine and replacement fans can be obtained by [contacting support](https://support.system76.com).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fans/heatsink/thermal paste:
+
+1. Power off the machine. Wait 20 minutes for the machine to cool down to room temperature.
+    - **Warning:** Removing the heatsink before the machine has cooled to room temperature may result in heat-conductive metal flowing onto the motherboard and causing irrepairable damage.
+2. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+3. Unscrew the fan and heatsink screws in the order of the stamped numbers, starting with #1, then #2, and continuing until you have removed #12.
+    - The screws are held captive and will not come out of the fans/heatsink. The fan screws will click when they are fully loosened.
+    - The order for the 17" model is shown below; the screws are numbered differently on the 15" model. Follow the order stamped into the heatsink metal.
+    - Do not remove the smaller screws holding the fan covers onto the fans.
+
+![Thermal screws](./img/thermal-screws.webp)
+
+4. If there is any clear tape securing the heatsink/fans or the fan wires, remove it.
+5. Unplug the two white fan connectors from the motherboard (highlighted red above).
+6. Remove the heatsink/fans from the case, being careful not to bend the heatsink pipes. It may take some pressure to break the seal of the thermal paste.
+7. Using a paper towel, remove the existing thermal paste from the integrated CPU die, the border of the CPU chip, the CPU heatsink, the border of the GPU die, and the GPU heatsink.
+    - A small amount of rubbing alcohol can be used if the old paste is dried or difficult to remove.
+    - The heat-conductive metal may have some holes where pieces stuck to the die or heatsink.
+    - If you are not replacing the heat-conductive metal, do not pick or wipe the heat-conductive metal off of the main CPU die or the GPU heatsink.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. If you are replacing the heat-conductive metal, remove the plastic covers from the GPU heatsink and the CPU chip.
+    - A fingernail or a thin plastic tool can be used to pick a side of the plastic cover up from the CPU chip/GPU heatsink.
+    - After removing the plastic covers, wipe away any remaining heat-conductive metal residue from the heatsink, dedicated GPU die, and main CPU die.
+    - Some adhesive residue may remain on the GPU heatsink and CPU chip.
+
+![Heat-conductive metal removal](./img/heat-conductive-metal-removal.webp)
+
+9. Apply the new heat-conductive metal to the CPU chip and the GPU heatsink, matching the orientation of the plastic coverings with the diagonal corners of their respective surfaces.
```

Comment:
>Apply the new heat-conductive metal to the CPU chip and the GPU heatsink

Shouldn't we advise that they put the metal on the GPU itself rather than the heatsink? Otherwise it'd be difficult to hold onto the heatsink while putting it back together, and it could be misaligned so that the GPU die doesn't sit in the cutout on the plastic part.

Reply:
No, the metal comes off with the heatsink when removing the heatsink and the guides to place the metal/plastic cover are located on the heatsink for the GPU.

Reply:
To clarify, there is only adhesive on one side of the plastic cover. That side needs to be facing the heatsink in order to be in the correct orientation. So it needs to be stuck onto the heatsink.

Reply:
Ah I gotcha, that makes sense. I misunderstood how that worked.